platform/posix/transport/src/openssl_posix.c

```diff
@@ -726,9 +726,13 @@ int32_t Openssl_Recv( NetworkContext_t * pNetworkContext,
         {
             sslError = SSL_get_error( pOpensslParams->pSsl, bytesReceived );
 
-            if( sslError == SSL_ERROR_WANT_READ )
+            if( ( sslError == SSL_ERROR_WANT_READ ) || ( sslError == SSL_ERROR_WANT_WRITE ) )
```

Comment:
This is blocking, so we don't expect to get `SSL_ERROR_WANT_WRITE`. Only need to check `SSL_ERROR_WANT_READ` here.

Reply:
Removed check for `SSL_ERROR_WANT_WRITE`